wolfssl/wolfcrypt/wolfmath.h

```diff
@@ -62,11 +62,9 @@ MP_API int get_rand_digit(WC_RNG* rng, mp_digit* d);
 WOLFSSL_API int mp_cond_copy(mp_int* a, int copy, mp_int* b);
 WOLFSSL_API int mp_rand(mp_int* a, int digits, WC_RNG* rng);
 
-enum {
-    /* format type */
-    WC_TYPE_HEX_STR = 1,
-    WC_TYPE_UNSIGNED_BIN = 2,
-};
+#define WC_TYPE_HEX_STR 1
+#define WC_TYPE_UNSIGNED_BIN 2
+#define WC_TYPE_BLACK_KEY 3
```

Comment:
Can we wrap `WC_TYPE_BLACK_KEY` in a CAMM macro so people know its not used for anything else?